drizzlepac/hapsequencer.py

```diff
@@ -334,6 +335,23 @@ def run_hla_processing(input_filename, result=None, debug=False):
         # instrument_programID_obsetID_manifest.txt (e.g.,wfc3_b46_06_manifest.txt)
         manifest_name = total_list[0].manifest_name
 
+        # Set up all pipeline parameter values that will be used later on in the run.
+        # First, build temp.list of all product objects
+        full_product_list = total_list.copy()
```

Comment:
You should work directly on the total_list.  This will allow the newly added '.pars' attribute from line 350 to become available to the rest of the steps.  Each step can then use the '.pars.get_pars('<processing step>')' syntax to retrieve the parameters for the processing step on-the-fly without any additional file I/O or bookkeeping for cfg filenames.

Reply:
Resolved by commit [33fc25c](https://github.com/spacetelescope/drizzlepac/pull/439/commits/33fc25c59307379fe90008fea1e7de121c4f0c93)